vllm_spyre/v1/core/scheduler.py

```diff
@@ -544,18 +544,22 @@ def can_schedule(self, request):
         # - prompt_len < self.tkv and fall into different blocks
         prompt_len = request.num_prompt_tokens
         n_blocks = math.floor(max(self.tkv, prompt_len) / self.block_size)
-        max_tkv = n_blocks * self.block_size + max(
-            self.tkv % self.block_size, prompt_len % self.block_size)
         new_tkv = n_blocks * self.block_size + prompt_len % self.block_size
```

Comment:
If I understand correctly, this tkv won't be the "new tkv" (ie, the next self.tkv value) but the tkv of the new joining prompt specifically. Since we don't have a single global and absolute tkv value anymore, and this is quite a major concept change, I would try to be very specific in the naming of the tkv values, to help and assimilate the new understanding, that makes sense.

```suggestion
        new_req_tkv = n_blocks * self.block_size + prompt_len % self.block_size
```

Reply:
if I change it here, the variables mean different things in continuous batching vs chunked prefill... which is bad. 
changing the CB variable names as part of this PR might also be a bit confusing as this is a pr about CP. 
I would probably keep CP var names consistent with CB var names here, and if needed address with a variable change in a follow up PR for both CP and CB. WDYT @sducouedic ?